### DISCUSSION TITLE
On whether the use of LGPL protocol is in compliance

### DISCUSSION THREAD

**Discussion by lc707776289** [mapsforge/vtm#1095] [Q&A] [answered]
~~~
dependencies {
def  map_version = "0.20.0"
    implementation "org.mapsforge:vtm:${map_version}"
    implementation "org.mapsforge:vtm-themes:${map_version}"
    implementation "org.mapsforge:vtm-http:${map_version}"
    /*implementation 'org.slf4j:slf4j-api:1.7.36'
    implementation 'org.slf4j:slf4j-android:1.7.36'*/

    runtimeOnly "org.mapsforge:vtm-android:${map_version}:natives-armeabi-v7a"
    runtimeOnly "org.mapsforge:vtm-android:${map_version}:natives-arm64-v8a"
    implementation "org.mapsforge:vtm-android:${map_version}"
}
~~~
In this way, dependency is introduced under Gradle, and can the software be used for commercial use?

**Comment by devemux86** [comment, marked as answer]
See here:
- https://github.com/mapsforge/vtm#license

**Reply by lc707776289** [reply]
I understand the LGPL agreement, if you want to commercial use but do not open source your own software, you need to use dynamic links, but the Gradle under Android using the way I have introduced should be static links?

**Reply by devemux86** [reply]
Please read the notice in https://github.com/mapsforge/vtm#license,
it contains a change to the license and explains that it can be used commercially on Android.
